tests/test_config_model.py

```diff
@@ -11,6 +11,40 @@ def test_pydantic_model(self):
             CONFIG_MODEL.zones["US-NW-PSCO"].parsers.get_function("production")
         )
 
+    # Add well-known sources that don't require config-based references here
+    GLOBAL_SOURCE_REFERENCES = {
+        "BEIS 2021",
+        "CEA 2022",
+        "EIA 2020/BEIS 2021",
+        "EU-ETS 2021",
+        "EU-ETS, ENTSO-E 2021",
+        "IEA 2019",
+        "IEA 2020",
+        'Oberschelp, Christopher, et al. "Global emission hotspots of coal power generation."',
+    }
+
+    def test_zone_sources(self):
+        for measurement_basis, model in CO2EQ_CONFIG_MODEL:
```

Comment:
Since measurement_basis is not used I would underscore it.

Reply:
Ah, my mistake committing a partial change-in-progress there: I was considering adding that variable to the [error message here](https://github.com/jayaddison/electricitymaps-contrib/blob/42e27fe69891c3d39c2e815d31c1fbd41198ed14/tests/test_config_model.py#L36) to help people figure out where the source entry is missing.  What do you think?

Reply:
Hopefully resolved; I think that making the error messages from this unit test as unambiguous as possible would make sense, so I've standardized the format of them a bit, and included the full path to the `source` key when it's missing (making use of the `measurement_basis` variable).